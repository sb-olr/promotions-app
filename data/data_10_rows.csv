2d522e2c-b2ac-4696-94b9-7dc95cd80ce6,21.591581,2018-06-27 10:05:45 +0200 CEST
878c4e7e-509e-41bd-8523-d24f9aff4a7b,97.468114,2018-06-12 06:55:16 +0200 CEST
5826b23c-949b-4e61-8910-4e70e6c70f59,69.322593,2018-06-20 00:03:00 +0200 CEST
0741c1f2-ac95-40df-ba39-7a1dea827955,41.061457,2018-07-06 20:15:12 +0200 CEST
5e0fca24-6b83-491c-84a3-7f0813719d19,74.660479,2018-08-22 22:10:19 +0200 CEST
60e56579-0028-4250-800a-151ef57dc7fc,44.119993,2018-07-21 23:18:57 +0200 CEST
afae7121-e922-4a89-adcc-a9717b1f23e8,24.023492,2018-10-15 09:08:11 +0200 CEST
db808869-1674-4651-af5a-a1e40afc57f6,29.195418,2018-08-29 14:17:47 +0200 CEST
0464518f-848f-4845-af64-5fb45a54ddd0,62.454782,2018-07-25 11:56:25 +0200 CEST
d431e6be-7538-49ab-bd94-cfd297c66b8e,9.689842,2018-06-04 06:01:20 +0200 CEST
