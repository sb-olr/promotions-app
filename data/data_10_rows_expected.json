[{"id":"2d522e2c-b2ac-4696-94b9-7dc95cd80ce6","price":"21.591581","expiration_date":"2018-06-27 10:05:45 +0200 CEST"},{"id":"878c4e7e-509e-41bd-8523-d24f9aff4a7b","price":"97.468114","expiration_date":"2018-06-12 06:55:16 +0200 CEST"},{"id":"5826b23c-949b-4e61-8910-4e70e6c70f59","price":"69.322593","expiration_date":"2018-06-20 00:03:00 +0200 CEST"},{"id":"0741c1f2-ac95-40df-ba39-7a1dea827955","price":"41.061457","expiration_date":"2018-07-06 20:15:12 +0200 CEST"},{"id":"5e0fca24-6b83-491c-84a3-7f0813719d19","price":"74.660479","expiration_date":"2018-08-22 22:10:19 +0200 CEST"},{"id":"60e56579-0028-4250-800a-151ef57dc7fc","price":"44.119993","expiration_date":"2018-07-21 23:18:57 +0200 CEST"},{"id":"afae7121-e922-4a89-adcc-a9717b1f23e8","price":"24.023492","expiration_date":"2018-10-15 09:08:11 +0200 CEST"},{"id":"db808869-1674-4651-af5a-a1e40afc57f6","price":"29.195418","expiration_date":"2018-08-29 14:17:47 +0200 CEST"},{"id":"0464518f-848f-4845-af64-5fb45a54ddd0","price":"62.454782","expiration_date":"2018-07-25 11:56:25 +0200 CEST"},{"id":"d431e6be-7538-49ab-bd94-cfd297c66b8e","price":"9.689842","expiration_date":"2018-06-04 06:01:20 +0200 CEST"}]
